app/src/main/java/ch/epfl/polychef/users/User.java

```diff
@@ -60,6 +68,18 @@ public User() {
         favourites = new ArrayList<>();
         subscribers = new ArrayList<>();
         subscriptions = new ArrayList<>();
+        rating = new Rating();
+    }
+
+    public User(String email, String username, Rating rating){
```

Comment:
Is it for database that we added this constructor ? Or for testing purposes ?

Reply:
database